CHANGELOG.md

```diff
@@ -35,11 +35,14 @@ Let's welcome [**@giltayar**](https://github.com/giltayar) and [**@nicojs**](htt
 - [#4223](https://github.com/mochajs/mocha/issues/4223): The context object's `skip()` (`this.skip()`) in a "before all" (`before()`) hook will no longer execute subsequent sibling hooks, in addition to hooks in child suites ([**@juergba**](https://github.com/juergba))
 
 - [#4178](https://github.com/mochajs/mocha/issues/4178): Remove previously soft-deprecated APIs ([**@wnghdcjfe**](https://github.com/wnghdcjfe)):
+
   - `Mocha.prototype.ignoreLeaks()`
   - `Mocha.prototype.useColors()`
   - `Mocha.prototype.useInlineDiffs()`
   - `Mocha.prototype.hideDiff()`
 
+- [#4200](https://github.com/mochajs/mocha/issues/4200): Drop mkdirp and replace it with fs.mkdirSync ([**@HyunSangHan**](https://github.com/HyunSangHan))
```

Comment:
This PR is not really a breaking change?
It's `semver-major` only because it isn't supported in Node v8.

Reply:
Agreed. "Enhancement" is better?

Reply:
I think "other".